### DISCUSSION TITLE
QEMU error: QEMU exited from an error: Operation not permitted

### DISCUSSION THREAD

**Discussion by mattweiman** [utmapp/UTM#5837] [UTM Help (macOS)] [answered]
**Describe the issue**  
I have a Windows 10 virtual machine on my Mac, been using it for a few years just fine. Used it yesterday as normal, no issues or anything. For some reason, when I'm trying to run it, I am suddenly getting this error:

"QEMU error: QEMU exited from an error: qemu-aarch64-softmmu: -drive if=none,media=cdrom,id=driveEEC5A6F8-6F8B-46A4-98C6-1DAE1CF2597B,file=/Users/mattweiman/Downloads/ISOs/WINDOWS.ISO,readonly=on: Could not open '/Users/mattweiman/Downloads/ISOs/WINDOWS.ISO': Operation not permitted"

I have not changed anything, let alone anything related to this VM or UTM at all. The file is definitely not readonly, it is and always has been read and write. I'm confused and a bit frustrated as I commonly use the VM for coursework and have a lab due soon with all my data on there. I have tried the typically turn it off and on again, turn my computer off and on again, edit the VM and click save, etc. Does anyone have any ideas for how to fix this?

**Configuration**  

* UTM Version: 4.4.4 (93)
* macOS Version: Ventura 13.4
* Mac Chip (Intel, M1, ...): M1

**Crash log**  
N/A

**Debug log**  
[debug.log](https://github.com/utmapp/UTM/files/13193047/debug.log)

**Upload VM**  
[config.plist.zip](https://github.com/utmapp/UTM/files/13193054/config.plist.zip)

**Comment by osy** [comment, marked as answer]
Eject the iso. Probably something on your system broke the bookmark to it.

**Reply by mattweiman** [reply]
Ah, thank you. Much less strenuous than I was anticipating.

**Reply by uliloewi** [reply]
How to eject?

**Reply by omardjo** [reply]
go to CD/DVD option (you'll see it by default in the utm interface) then click clear

**Reply by zahsaps** [reply]
THANK YOU - THIS SAVED MY TIME.

**Reply by ziadh72** [reply]
what if it is my first time what do i do

**Reply by ziadh72** [reply]
do i also eject the iso or?

**Reply by a4kulshr44** [reply]
thanks! saved my time

**Reply by NourMaka** [reply]
Thank you! That helped :)

**Reply by matimacayal** [reply]
Thanks! Does someone knoe why does this work?

**Comment by raven1779** [comment]
@osy hello can you help me with this issue?

QEMU error: QEMU exited from an error: qemu-aarch64-softmmu: -drive if=none,media=cdrom,id=drive8DE59B3B-0696-425E-BF49-8AFA6361A371,file=/Users/hartanto/Downloads/22631.2861.231204-0538.23H2_NI_RELEASE_SVC_REFRESH_CLIENTCONSUMER_RET_A64FRE_en-us.iso,readonly=on: Failed to lock byte 100

**Reply by osy** [reply]
Please open a new issue and attach debug.log. Is the ISO open anywhere else?

**Reply by raven1779** [reply]
i finally found a solution, thanks.

But i do have another question, can i connect my phone and access it in UTM?

**Reply by osy** [reply]
Can you post what the solution is for other people?

**Reply by raven1779** [reply]
when you download the windows, sometimes you accidentally open the windows, try ejecting the windows file and try relaunching it. works well

**Comment by raven1779** [comment]
@osy Can i conncet my iphone to UTM? so i can access my phone through utm

**Reply by osy** [reply]
Unfortunately that seems to be broken
